docker-compose.yml

```diff
@@ -48,13 +46,13 @@ services:
       - '.:/var/www/html/wp-content/plugins/wp-graphql-acf'
       - './.log/testing:/var/log/apache2'
       - './codeception.dist.yml:/var/www/html/wp-content/plugins/wp-graphql-acf/codeception.yml'
-    env_file: .env.dist
+    env_file:
```

Comment:
Banged head on wall because this env_file stuff + environment vars + shell environment vars don't act in the cascading way you'd expect or how it's documented.